File: 74751_lab2_task1.cpp
// 74751_lab2_task1.cpp
#include <iostream>
#include <cmath>
#include <vector>

using namespace std;

int main() {
    vector<double> rr_intervals(6);
    for (int i = 0; i < 6; ++i) {
        cout << "Enter RR interval " << (i + 1) << " (seconds): ";
        cin >> rr_intervals[i];
    }
    
    int irregularity_count = 0;
    for (int i = 1; i < 6; ++i) {
        if (abs(rr_intervals[i] - rr_intervals[i-1]) > 0.2) {
            irregularity_count++;
        }
    }
    
    cout << "Irregularity count: " << irregularity_count << endl;
    if (irregularity_count > 2) {
        cout << "Potential arrhythmia flagged." << endl;
    }
    
    return 0;
}

File: 74751_lab2_task1.py
# 74751_lab2_task1.py
rr_intervals = []
for i in range(6):
    val = float(input(f"Enter RR interval {i+1} (seconds): "))
    rr_intervals.append(val)

irregularity_count = 0
for i in range(1, 6):
    if abs(rr_intervals[i] - rr_intervals[i-1]) > 0.2:
        irregularity_count += 1

print(f"Irregularity count: {irregularity_count}")
if irregularity_count > 2:
    print("Potential arrhythmia flagged.")
